src/pages/Home/Home.js

```diff
@@ -189,14 +189,17 @@ with Grommet and the Grommet Designer."
     <Box
       margin={{
         top: 'xlarge',
-        left: 'xlarge',
-        right: 'xlarge',
+        horizontal: 'xlarge',
         bottom: 'none',
       }}
-      direction="row"
+      direction="row-responsive"
+      align="end"
+      gap="xlarge"
     >
-      <Box width="medium" margin={{ bottom: 'xlarge' }}>
-        <Heading>What are grommet Tools?</Heading>
+      <Box alignSelf="center" size="large" margin={{ bottom: 'xlarge' }}>
+        <Heading level={2} size="xlarge">
```

Comment:
Can you confirm you have a Level 1 heading before applying the second?

Reply:
Added level 1 heading to 'Grommet Tools' at the top of the page and removed custom headings from theme